data-prepper-api/src/main/java/com/amazon/dataprepper/model/span/Link.java

```diff
@@ -0,0 +1,39 @@
+package com.amazon.dataprepper.model.span;
```

Comment:
Perhaps this package should be `trace` since we speak of this generally as "trace analytics." All models related to "trace analytics" would go in the `trace` package.